Dockerfile

```diff
@@ -9,10 +9,33 @@ LABEL about.license="MIT License (MIT)"
 # Install required libs
 RUN apk update
 RUN apk --no-cache add make automake gcc g++ linux-headers libffi-dev zlib-dev \
-   jpeg-dev libressl-dev cairo-dev pango-dev gdk-pixbuf ttf-freefont bash
+   jpeg-dev libressl-dev cairo-dev pango-dev gdk-pixbuf ttf-freefont bash \
+   autoconf musl-dev perl bzip2-dev xz-dev curl-dev
 RUN pip install numpy Cython
```

Comment:
Perhaps we could follow Codefactor's suggestion?

Reply:
Yes for apk, all hands down. I'm not quite sure about pip, but I suppose it will still use the previously installed ones anyway, so probably all good.